crypto/evp/pmeth_fn.c

```diff
@@ -114,6 +115,48 @@ static void *evp_signature_from_dispatch(int name_id,
                 = OSSL_get_OP_signature_verify_recover(fns);
             verifyrecfncnt++;
             break;
+        case OSSL_FUNC_SIGNATURE_DIGEST_SIGN_INIT:
+            if (signature->digest_sign_init != NULL)
+                break;
```

Comment:
Here are below, should duplicates like this be treated as errors, rather than ignored?

Reply:
This is how *all* the `evp_*_from_dispatch` functions work so I think we should be consistent here. I don't have a strong view either way about what is the correct strategy, but if we change strategy I think it should be in a separate PR.

Reply:
Cool.